plugins/obs-ffmpeg/obs-ffmpeg-source.c

```diff
@@ -316,6 +323,8 @@ static void ffmpeg_source_update(void *data, obs_data_t *settings)
 			"color_range");
 	s->buffering_mb = (int)obs_data_get_int(settings, "buffering_mb");
 	s->is_local_file = is_local_file;
+	s->is_not_seekable = obs_data_get_bool(settings,
```

Comment:
Style:  This line and the line below can be on one line without hitting the 80 column line length limit.